libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -767,7 +767,7 @@ void NavEKF2::UpdateFilter(void)
 
     // when disarmed force lane1 if it is healthy. This is a temporary
     // workaround for poor performance of the 2nd IMU
-    if (!hal.util->get_soft_armed() && primary != 0 && core[0].healthy()) {
+    if (!hal.util->get_soft_armed() && primary != 0) {
```

Comment:
Just want to make sure, now that it will always stay in 1st lane, even when it is unhealthy. The vehicle will not be able to pass the pre-arm checks in case lane 1 is unhealthy, right?
Because I don't want the vehicle to be capable of arming even when the EKF lane it is on is unhealthy.